custom/icds_reports/migrations/0038_aggregatesqlprofile.py

```diff
@@ -0,0 +1,22 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.11.12 on 2018-04-07 16:11
+from __future__ import unicode_literals
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('icds_reports', '0037_add_new_infra_data'),
+    ]
+
+    operations = [
+        migrations.CreateModel(
+            name='AggregateSQLProfile',
+            fields=[
+                ('id', models.AutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
+                ('time', models.DateTimeField()),
```

Comment:
This will need to be recreated (it doesn't have line_num). Also there were some new migrations since this was made

Reply:
Rebased and fixed.